### PR TITLE
distro/adaptation/centos: map fedora packages to centos

### PR DESCRIPTION
**Pull request by jiaqizhao122**

### CODE REVIEW

File: distro/adaptation/centos [hunk 1]
```diff
@@ -1,5 +1,6 @@
 apache2: httpd
 apt-file:
+arping: iputils
 autopoint: gettext-devel
```
Comment (rli9) [inline distro/adaptation/centos:4]: this mapping is different in fedora, which is autopoint: gettext. Need check whether centos can use same package here or not?

**Review by rli9 — COMMENTED** [review]
For subject, use distro/adaptation/centos: map packages a* from fedora

File: distro/adaptation/centos (hunk 1 above)
Comment (jiaqizhao122) [reply distro/adaptation/centos:4]: yes, i have checked this, both gettext and gettext-devel is avilable in centos789, so i keeped one that is already exist in centos.

**Comment by jiaqizhao122** [comment]
> For subject, use distro/adaptation/centos: map packages a* from fedora

ok

**Review by rli9 — COMMENTED** [review]
default-jdk mapping is not added.

File: distro/adaptation/centos [hunk 2]
```diff
@@ -12,9 +12,10 @@ cgroupfs-mount:
 clang-6.0: clang.x86_64
 cmake: cmake cmake3
 cron: cronie
-dbench=4.0-2: dbench
+dbench=4.0-2:
```
Comment (rli9) [inline distro/adaptation/centos:15]: let's keep dbench mapping for now, it probably uses lkp-tests/programs/dbench

**Comment by jiaqizhao122** [comment]
> default-jdk mapping is not added.
but i have added "default-jdk: java-1.8.0-openjdk-devel" in centos, is that not correct?

File: distro/adaptation/centos (hunk 2 above)
Comment (jiaqizhao122) [reply distro/adaptation/centos:15]: ok!

**Comment by rli9** [comment]
thanks for the patch

**Pull request merged**